### ISSUE TITLE
Keep one space from form to inline comment

### ISSUE THREAD

**Issue by arichiardi** [kkinnear/zprint#191]
Hi Kim, I have been noticing that `zprint` is making this change to inline comments and I wonder if there is an option for keeping that separation between the form and the comment.

Basically I would like to keep the first line of the following diff:
```
-(s/def ::graphical-scoring ::legacy-tree) ;; just so you know
+(s/def ::graphical-scoring ::legacy-tree);; just so you know
```

Thanks - It is minor but I thought it would be good to report it.

**Comment by kkinnear** [comment]
Thanks for noticing this.  I have also recently noticed something probably related to this as well.  Inline comments are supposed to preserve the spacing from code, modulo preserving their interline alignment.   So I think they shouldn't be doing this.

Seems like inline comments work well when then are inside of expressions, but not so well when they are at the "top level".  I will work on this, but the fix won't be in `1.1.2`, as I am trying really hard to get that out soon.

**Comment by kkinnear** [comment]
This was fixed in `1.2.0` last November.
